web-testbed/tests/tests_backend/proxies/non_widget_proxy.py

```diff
@@ -0,0 +1,17 @@
+from .base_proxy import BaseProxy
+
+
+class NonWidgetProxy(BaseProxy):
+    _storage_expr = "self.my_objs"
+    # _ctor_expr: str | None = None
+
+    def _create(self, ctor_expr: str, *args, **kwargs) -> str:
+        call_args = self._encode_call(*args, **kwargs)
+        code = (
+            "import uuid\n"
+            f"new_obj = {ctor_expr}({call_args})\n"
+            "key = str(uuid.uuid4())\n"
```

Comment:
There's no need to invent a new unique ID for an object. `id(myobj)` will return a unique integer identifier.